lib/secure-random-string.js

```diff
@@ -26,7 +26,11 @@ function srs(options, cb) {
 
 	function _finish (buf) {
 		var string = buf.toString('base64');
-		string = string.replace(/\//g,'_').replace(/\+/g,'-');
+		if (alphanumeric === true){
```

Comment:
please insert a whitespace between the `)` and the `{`
`if (alphanumeric === true) {`